internal/cmd/schema-registry/command_cluster_update.go

```diff
@@ -68,21 +72,72 @@ func (c *command) updateTopLevelCompatibility(cmd *cobra.Command) error {
 		return err
 	}
 
-	compatibility, err := cmd.Flags().GetString("compatibility")
+	updateReq, err := c.getConfigUpdateRequest(cmd)
 	if err != nil {
 		return err
 	}
 
-	updateReq := srsdk.ConfigUpdateRequest{Compatibility: strings.ToUpper(compatibility)}
-
-	if _, _, err := srClient.DefaultApi.UpdateTopLevelConfig(ctx, updateReq); err != nil {
+	if _, _, err := srClient.DefaultApi.UpdateTopLevelConfig(ctx, *updateReq); err != nil {
 		return err
 	}
 
 	output.Printf(errors.UpdatedToLevelCompatibilityMsg, updateReq.Compatibility)
 	return nil
 }
 
+func (c *command) getConfigUpdateRequest(cmd *cobra.Command) (*srsdk.ConfigUpdateRequest, error) {
```

Comment:
Without pointer `srsdk.ConfigUpdateRequest` I cannot return `nil` if there's error, how does people usually do it in golang?

Reply:
Typically you'd want to return the zero value (`0`, `""`, `nil`, etc.). In this case, this would be the zero value:
```
return srsdk.ConfigUpdateRequest{}, err
```